internal/cmd/cmd.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/sample"
+	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/util"
+	"github.com/spf13/cobra"
+	"log"
+	"os/exec"
+	"path/filepath"
+)
+
+// Root is responsible for the root command. It handles the application flow.
+func Root(cmd *cobra.Command, args []string) error {
+	// Parse sample directory from command line argument
+	sampleDir, err := filepath.Abs(filepath.Dir(args[0]))
+	if err != nil {
+		return err
+	}
+
+	log.Println("Setting up configuration values")
+	s, err := sample.NewSample(sampleDir)
+	if err != nil {
+		return err
+	}
+
+	log.Println("Loading test endpoints")
+	swagger := util.LoadTestEndpoints()
+
+	log.Println("Building and deploying sample to Cloud Run")
+	err = s.BuildDeployLifecycle.Execute(s.Dir)
+	defer s.Service.Delete(s.Dir)
+	defer s.DeleteCloudContainerImage()
+	if err != nil {
+		return fmt.Errorf("[cmd.Root] building and deploying sample to Cloud Run: %w", err)
+	}
+
+	log.Println("Getting identity token for gcloud auhtorized account")
+	var identToken string
+
+	a := append(util.GcloudCommonFlags, []string{
+		"auth",
+		"print-identity-token",
+	}...)
```

Comment:
The variadic operator and slice cancel each other out.  
```suggestion
	a := append(util.GcloudCommonFlags, "auth", "print-identity-token")
```

Reply:
you do this in a few other places as well